readme.md

```diff
@@ -543,3 +575,22 @@ To query the list of the time-based-inflation entries in the tokenomics module,
 ```
 elysd q tokenomics list-time-based-inflation
 ```
+
+## Release
+
+To release a new version of your blockchain, create and push a new tag with `v` prefix. A new draft release with the configured targets will be created.
+
+```
+git tag v0.1
+git push origin v0.1
+```
+
+After a draft release is created, make your final changes from the release page and publish it.
+
+## Learn more
+
+- [Ignite CLI](https://ignite.com/cli)
```

Comment:
Do we need to put Ignite CLI document on elys repo?

Reply:
I want to keep it here until we add our own Makefile